### PR TITLE
Translations for filter.252, tags.65 and tags.66

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/FiltersRegistry/issues/593

### CODE REVIEW

File: locales/en/filters.json [hunk 1]
```diff
@@ -338,5 +338,9 @@
 	{
 		"filter.251.name": "Legitimate URL Shortener",
 		"filter.251.description": "Automatically removes unnecessary '$' and '&' values from URLs, making them easier to copy from the URL bar and pasting elsewhere as links."
+	},
+	{
+		"filter.252.name": "Dandelion Sprout's Serbo-Croatian List",
+		"filter.252.description": "For the first time in known history, this is a filterlist for websites in Serbian, Montenegrin, Croatian and Bosnian. This means that all significant language groups in Europe now have filterlists for them."
```
Comment (slavaleleka) [inline locales/en/filters.json:344]: второе предложение как-то выбивается

Comment (el-termikael) [reply locales/en/filters.json:344]: да и For the first time in known history вначале как-то пафосно
может, просто: This is a filterlist for websites in Serbian, Montenegrin, Croatian, and Bosnian?

Comment (el-termikael) [reply locales/en/filters.json:344]: да и For the first time in known history вначале как-то пафосно
может, просто: This is a filterlist for websites in Serbian, Montenegrin, Croatian, and Bosnian?

**Review by el-termikael — CHANGES_REQUESTED** [review]

File: locales/en/filters.json (hunk 1 above)
Comment (zloyden) [reply locales/en/filters.json:344]: Ага, поменяю.

**Review by slavaleleka — APPROVED** [review]

**Review by el-termikael — APPROVED** [review]

**Comment by zloyden** [comment]
Do not merge, waiting for translations.

**Comment by ameshkov** [comment]
@zloyden what about now?

**Comment by zloyden** [comment]
They are ready now, I will add them here.

**Review by el-termikael — APPROVED** [review]

**Pull request merged**